files/kubelet.service

```diff
@@ -13,7 +13,7 @@ ExecStart=/usr/bin/kubelet --cloud-provider aws \
     --network-plugin cni $KUBELET_ARGS $KUBELET_EXTRA_ARGS
 
 Restart=on-failure
```

Comment:
Let's instead change this to `Restart=always`.

Reply:
Sounds good

Reply:
I have also updated the PR title accordingly